Products/ZenEvents/events2/processing.py

```diff
@@ -68,9 +70,11 @@ class Manager(object):
         COMPONENT: DeviceComponent,
     }
 
-    def __init__(self, dmd):
+    def __init__(self, dmd, CONF_LOG_PERF_AS_INFO=False, CONF_slowSegmentThreshold=1.0):
```

Comment:
Stylistically, change these argument names to camelCase.  We don't really need the `CONF_` prefix either.